apps/trading/client-pages/markets/top-market-list.tsx

```diff
@@ -24,14 +25,22 @@ export const TopMarketList = ({
             key={market.id}
           >
             <span className="col-span-3 overflow-hidden">
-              <Link to={Links.MARKET(market.id)}>
-                <span className="flex items-center">
-                  <span className="pr-1">
-                    <EmblemByMarket market={market.id} vegaChain={chainId} />
+              <Tooltip description={market.tradableInstrument.instrument.code}>
```

Comment:
because there is not enough room for the full market code for certain markets, they'll be displayed with ellipsis, and you could hover over to see the full code.

Reply:
<img width="1063" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/8245452b-312b-47aa-bc96-4aa799f32e76">
 not great but we're trying to fit in too much over there

Reply:
I can put instead the market name

Reply:
ok, updated with the market name